src/jobs/models/kubernetes/jobConnector.js

```diff
@@ -52,16 +52,52 @@ module.exports.stopRun = async (jobPlatformName, platformSpecificInternalRunId)
     await requestSender.send(options);
 };
 
-module.exports.getLogs = async (jobPlatformName, platformSpecificInternalRunId) => {
+module.exports.getLogs = async (jobPlatformName, platformSpecificInternalRunId, predatorRunnerPrefix) => {
     let jobControllerUid = await getJobControllerUid(jobPlatformName, platformSpecificInternalRunId);
 
     let podsNames = await getPodsByLabel(jobControllerUid);
 
-    let logs = await getLogsByPodsNames(podsNames);
+    let logs = await getLogsByPodsNames(podsNames, predatorRunnerPrefix);
 
     return logs;
 };
 
+module.exports.deleteAllContainers = async (jobPlatformName) => {
+    let jobs = await getAllPredatorRunnerJobs(jobPlatformName);
+    let allPredatorRunnersPods = [];
+    for (let i = 0; i < jobs.length; i++) {
+        const pods = await getPodsByLabel(jobs[i].metadata.uid);
+        allPredatorRunnersPods = allPredatorRunnersPods.concat(pods);
+    }
+
+    let deleted = 0;
+    for (let i = 0; i < allPredatorRunnersPods.length; i++) {
+        let pod = await getPodByName(allPredatorRunnersPods[i]);
+
+        let containers = pod.status.containerStatuses;
+        containers = containers.find(o => o.name === 'predator-runner');
```

Comment:
`predator-runner` is used twice in this file, use it as a const

`const PREDATOR_RUNNER = 'predator-runner'`